src/main/java/duke/action/Todo.java

```diff
@@ -0,0 +1,54 @@
+package duke.action;
+
+/**
+ * An extension from Action.
+ */
+public class Todo  extends Action {
+
+    /**
+     * Constructs a new Todo object with a String variable s.
+     * @param s task todo
+     */
+    public Todo(String s)  {
+        super(s);
+    }
+
+    /**
+     * Constructs a new Todo object with a String variable s,
+     * and a boolean variable, bool.
+     * @param s task todo
+     * @param bool done status of task todo
+     */
+    public Todo(String s, boolean bool)  {
+        super(s, bool);
+    }
+
+    /**
+     * Returns a new Todo object with same variable values except
+     * isDone which is now true
+     * @return marked Todo object
+     */
+    @Override
+    public Action setDone()  {
```

Comment:
```suggestion
    public Action setDone() {
```
Coding standard spacing